### ISSUE TITLE
May I offer a Chinese translation ?

### ISSUE THREAD

**Issue by zhenizhui** [swisnl/jQuery-contextMenu#572]
Hello, buddy ! This is a nice plugin which help me to finish my work quickly. 

I would like to do something to help more Chinese Developer to know this plugin and I really want to know how it works at the same time. 😄 

So, may I offer a Chinese translation? I'll do my best , although I haven't been learning javascript for a long time.

**Comment by bbrala** [comment]
Yeah that would be pretty awesome. How would you do that? Just translate the docs? I could just generate a CN version also if you just create a copy of the folder `documentation` and call is `documentation_cn` for now. We could worry about the proper structure later.

What do you think?

**Comment by zhenizhui** [comment]
Sound great! Yeah, just translate the docs at present And I will start to translate as you suggest.

**Comment by zhenizhui** [comment]
Sound great! Yeah, just translate the docs at present And I will start to translate as you suggest.

If I have any questions, may I ask you for advice through email? 😄

**Comment by bbrala** [comment]
Yeah no problem, best address is probably bjorn@swis.nl for this.

Op 3 okt. 2017 11:20 schreef "zhenizhui" <notifications@github.com>:

**Comment by bbrala** [comment]
Hi, 

Have you been working on this? I will be releasing 3.0 soon, which means the location of the documentation will change. The content is mostly the same though.

Any update?

**Comment by zhenizhui** [comment]
@bbrala 
hi, i still stay focused on this repository. 

But the progress is slow because of the busy work. 

Sorry about that. 😞

**Comment by bbrala** [comment]
Glad to hear from you. 

I think most of the standard docs are the same for 3.x and there are still a few I forgot to port to 3.x. I think moving over to 3.x won't be hard, but perhaps it's smart to wait a few weeks for me to get the documentation finished for 3.x.

Don't worry about it taking a while, I just wanted to know if there was still an intention to finish it.

**Comment by ChenViVi** [comment]
@zhenizhui
I'm using it on my project,too. So how is your progress, may I join this translation plan?

**Comment by zhenizhui** [comment]
@ChenViVi 
Yeah, welcome!

please send your QQ to this email address zhuinizhe@gmail.com. Then we can communicate more conveniently. 😃

**Comment by ChenViVi** [comment]
@bbrala  In the meanwhile, I also find something don't make sense to me. Maybe it's right to write "promise" as "promis" and "swype" is a word, it's just I don't know it. Whatever, have a look at the images below.

![](http://omqwegju8.bkt.clouddn.com/18-4-25/6175324.jpg)

![](http://omqwegju8.bkt.clouddn.com/18-4-25/24290745.jpg)

![](http://omqwegju8.bkt.clouddn.com/18-4-25/8329851.jpg)

![](http://omqwegju8.bkt.clouddn.com/18-4-25/31499706.jpg)

**Comment by bbrala** [comment]
Hmm, i think we should just remove the touchswipe page then. I'll make a new issue for the typo in promise. 

For the branding, i think ill just hide it on the mobile view, that seems like the easiest fix :)
